svthalia/concrexit
# 3325 Handle Invalid Filter Values

Closes #3325

### Summary
Bug: https://thalia.sentry.io/issues/4448548023/?project=1463433&query=is%3Aunresolved+issue.category%3Aerror&referrer=issue-stream&stream_index=8

Instead of a 500 error, it should now through a 400 ValidationError.

### How to test
Steps to test the changes you made:
1. Go to http://staging.thalia.nu/api/v2/admin/events/183/registrations/
2. Apply a filter that in the list: "y", "yes", "t", "true", "on", "1"; or "n", "no", "f", "false", "off", "0".
3. Page should throw a 400 error.
